### ISSUE TITLE
Support ?: expression

### ISSUE THREAD

**Issue by charlieMonroe** [drkameleon/iswift#96]
In ObjC/C (C99+), this is a valid expression, however, iSwift doesn't recognize it:

```
NSString *epString = data2[@"ep"] ?: @"";
```

The Swift equivalent is:

```
let epString = data2["ep"] ?? ""
```

**Comment by drkameleon** [comment]
That's true - it has not been implemented yet. I must have totally forgotten about it - and the weird things is I myself I'm making some *extensive* use of the ternary operator.

**Comment by drkameleon** [comment]
Re: https://developer.apple.com/library/ios/documentation/Swift/Conceptual/Swift_Programming_Language/BasicOperators.html#//apple_ref/doc/uid/TP40014097-CH6-ID72

**Comment by drkameleon** [comment]
Issue fixed as of the upcoming 2.0.
